### ISSUE TITLE
Feature: Get line(s) from jupyter console

### ISSUE THREAD

**Issue by fenuks** [jupyter-vim/jupyter-vim#99]
A clear and concise description of:
1. What would be nice to have
Option to load into vim buffer lines run in jupyter console
2. Why it must be added to the plugin
Let's say I loaded my file into console because I don't know how to use some module. I try several ways, and I finally figured out how to proceed. I would like to copy what I typed in jupyter console back into vim buffer. Jupyter allows to rerun selected lines e.g.

```
In [5]: print(5)
5

In [6]: %rerun 5
=== Executing: ===
print(5)
=== Output: ===
5
```

I would like to be able to type `:JupyterGetCell 5` to load this line into vim. Line number specification should be flexible and allow ranges (`:JupyterGetCell 5-10`), and several line numbers (`:JupyterGetCell 5-8 10`).

**Comment by wmvanvliet** [comment]
I like this idea. This is contingent on whether the jupyter kernel itself keeps a history of cell input and output.
